examples/videocomposer/README.md

```diff
@@ -2,6 +2,129 @@
 
 MindSpore implementation & optimization of [VideoComposer: Compositional Video Synthesis with Motion Controllability](https://arxiv.org/pdf/2306.02018.pdf).
 
+## Gallery
+
+### Inference different conditions from a video
+
+<div align="center">
+<img src="https://github.com/HaoyangLee/mindone/assets/20376974/d073e00f-d72f-4d69-9670-5685ff3d25b5" width="720" />
+</div>
+<p align="center">
+<em> Condition: image depth <br /> Text input: "A black swan swam in the water" </em>
+</p>
+
+<br />
+<div align="center">
+<img src="https://github.com/HaoyangLee/mindone/assets/20376974/a36ea362-d697-41e1-a8e6-7c80d1d5cd45" width="720" />
+</div>
+<p align="center">
+<em> Condition: local image <br /> Text input: "A black swan swam in the water" </em>
+</p>
+
+<br />
+<div align="center">
+<img src="https://github.com/HaoyangLee/mindone/assets/20376974/6e4d51fc-d6c0-43fe-9107-10bd258c10d7" width="720" />
+</div>
+<p align="center">
+<em> Condition: mask <br /> Text input: "A black swan swam in the water" </em>
+</p>
+
+<br />
+<div align="center">
+<img src="https://github.com/HaoyangLee/mindone/assets/20376974/afa49b6c-be68-4454-aa78-950674e3ec35" width="720" />
+</div>
+<p align="center">
+<em> Condition: motion <br /> Text input: "A black swan swam in the water" </em>
+</p>
+
+<br />
+<div align="center">
+<img src="https://github.com/HaoyangLee/mindone/assets/20376974/cdaabe13-3d28-4a25-bc00-1dc86e6507f3" width="720" />
+</div>
+<p align="center">
+<em> Condition: sketch <br /> Text input: "A black swan swam in the water" </em>
+</p>
+
+### Motion Transfer from a video to a Single Image
```

Comment:
The title capitalization can be improved, e.g., Motion Transfer from a video to a Single Image -> Motion Transfer From a Video to a Single Image. Pls check some other titles using https://capitalizemytitle.com

Reply:
fixed